PrizeDrawWeb/Services/Meetup/MeetupUrlParser.cs

```diff
@@ -0,0 +1,21 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace PrizeDrawWeb.Services.Meetup
+{
+    public static class MeetupUrlParser
+    {
+        public static (string, string) Parse (string url)
+            {
+             if ( Uri.TryCreate(url,UriKind.Absolute, out var address))
+            {
+                var group= address.Segments[1].TrimEnd('/');
```

Comment:
Can you add some verification that this is a meetup url for an event? Currently it would be possible to 'successfully' parse a totally unrelated URL that has sufficient segments in the address. I'd prefer to catch that at the point you enter the URL rather than continue and get an error later

Reply:
ah yes good point!

Reply:
Maybe verify the host is meetup.com and the segment before the id is `events`?